radiomics/imageoperations.py

```diff
@@ -745,9 +745,9 @@ def _swt3(inputImage, wavelet='coif1', level=1, start_level=0, axes=(2, 1, 0)):
     raise ValueError('Expected 3D data array')
 
   original_shape = matrix.shape
-  adjusted_shape = tuple([dim + 1 if dim % 2 != 0 else dim for dim in original_shape])
+  padding = tuple([(0, 1 if dim % 2 != 0 else 0) for dim in original_shape])
   data = matrix.copy()
-  data.resize(adjusted_shape, refcheck=False)
+  data = numpy.pad(data, padding, 'constant')
```

Comment:
As discussed, should change boudnary mode to `wrap`.

Reply:
done